sematic/api/server.py

```diff
@@ -76,6 +93,36 @@ def ping():
     return jsonify({"status": "ok"})
 
 
+def _request_string() -> str:
+    query_string = (
+        f"?{str(request.query_string, encoding='utf8')}"
```

Comment:
Please add `request` as a parameter to `_request_string`, so that only functions with `@sematic_api` magically refer to it, and use the `flask.request` spelling of it in those functions, to disambiguate.

Reply:
done